src/commands/moderation/temporaryCommandEdit.ts

```diff
@@ -0,0 +1,339 @@
+import { Colors, EmbedField, TextChannel } from 'discord.js';
+import moment from 'moment';
+import { CommandDefinition } from '../../lib/command';
+import { Roles, Channels, CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { getConn } from '../../lib/db';
+import TemporaryCommand from '../../lib/schemas/temporaryCommandSchema';
+
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MODERATION_TEAM,
+];
+
+const helpEmbed = (evokedCommand: String) => makeEmbed({
+    title: 'Manage Temporary Commands - Help',
+    description: 'A command to manage temporary commands that consist of a Title, Content and Color. This can be used to create temporary commands for different purposes.',
+    fields: [
+        {
+            name: 'Adding a Temporary Command',
+            value: makeLines([
+                'To create a temporary command, run the following bot command: ',
```

Comment:
```suggestion
                'To create a temporary command, use the following bot command: ',
```

Continuity if you agree with above again ;)